### PR TITLE
Replace collection for simplebook

### LINKED ISSUE #122: Add SimpleBook to a competition
SimpleBook is *very close* to being ready, and we're ready to make it live on a competition so we can see how it renders real data.

Since book printing doesn't work now anyway, we're feeling OK with putting it on an active competition, and I've picked LLIIA2020 out of a magic hat.

### PR DESCRIPTION
**Pull request by slifty**
This PR updates all competitions to use simplebook instead of collection.

If an existing deployment is using collection, the wiki configuration
must be manually updated (or explicitly replaced) so that both
simplebook and collection are not present at the same time

Resolves #122

### CODE REVIEW

**Comment by kfogel** [comment]
Minor comment on the preposition: the "for" confused me.  I expected "Replace Collection with SimpleBook" or "Replace Collection by SimpleBook" -- that way the directionality is unambiguous: we know what is replacing what.

I also lean toward writing the plugins' names as proper nouns, with CamelCase for "SimpleBook", especially given that one of them is a regular English word and the other one is the concatenation of two regular English words (and thus an uninformed reader might wonder if "simplebook" is just a typo for "simple book").

File: competitions/100Change2020/ansible/100Change2020.yml [hunk 1]
```diff
@@ -12,11 +12,11 @@
   roles:
     - mediawiki
     - activitylog
-    - collection
+    - simplebook
     - embed_video
     - permissions
     - simplefavorites
-    # PickSome has to come after collection so the left menu bar is correct
+    # PickSome has to come after simplebook so the left menu bar is correct
```
Comment (kfogel) [inline competitions/100Change2020/ansible/100Change2020.yml:19]: Okay, the original comment was odd too, in that it proper-noun'd "PickSome" but not "Collection".  I'd say let's follow the convention "PickSome" started and write "SimpleBook" here.

(Same in repeat occurrences below, of course.)

File: competitions/Climate2030/ansible/Climate2030.yml [hunk 2]
```diff
@@ -5,17 +5,17 @@
   become_user: "{{ deployment_user }}"
   roles:
     - mysql
-  
+
```
Comment (kfogel) [inline competitions/Climate2030/ansible/Climate2030.yml:8]: Hunh, this whitespace changed happened in most of the files.  It's not a big deal here, but I'm wondering if there's some editor setting you have turned on that is likely to cause more extensive non-substantive whitespace changes in the future depending on the state of the files edited.

Comment (slifty) [reply competitions/Climate2030/ansible/Climate2030.yml:8]: We have an .editorconfig that removes trailing whitespace -- the bug here is that the original author did not IMO!

Comment (frankduncan) [reply competitions/Climate2030/ansible/Climate2030.yml:8]: Hey now!

File: competitions/100Change2020/ansible/100Change2020.yml (hunk 1 above)
Comment (frankduncan) [reply competitions/100Change2020/ansible/100Change2020.yml]: Fixed!

**Pull request merged**